Modules/_cursesmodule.c

```diff
@@ -4853,29 +4835,73 @@ static PyMethodDef PyCurses_methods[] = {
     {NULL,                  NULL}         /* sentinel */
 };
 
-/* Initialization function for the module */
+/* Module C API */
 
+/* Function versions of the 3 functions for testing whether curses has been
+   initialised or not. */
 
-static struct PyModuleDef _cursesmodule = {
-    PyModuleDef_HEAD_INIT,
-    "_curses",
-    NULL,
-    -1,
-    PyCurses_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
+static inline int
+curses_capi_setupterm_called(void)
+{
+    return _PyCursesCheckFunction(curses_setupterm_called, "setupterm");
+}
+
+static inline int
+curses_capi_initscr_called(void)
+{
+    return _PyCursesCheckFunction(curses_initscr_called, "initscr");
+}
+
+static inline int
+curses_capi_start_color_called(void)
+{
+    return _PyCursesCheckFunction(curses_start_color_called, "start_color");
+}
+
+static void *
+_curses_capi_new(_cursesmodule_state *state)
```

Comment:
`_curses` is harder to type and read than `curses`. I don't think that it's worth it to be pedantic about the module name here, I prefer `curses`. Would you mind to rename these C API and Capsule functions to remove the `_` prefix?

Reply:
No I don't mind. I just did what I actually usually do but all extension modules just have their own convention as well, I can live with a less-pedantic-and-more-usable one.